net/eds/Makefile

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2019 NXP
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eds
+PKG_VERSION:=lsdk-1903
```

Comment:
tag is wrong: LSDK-19.03 is the correct one.

Reply:
Will change to LSDK-19.03. Thanks.